packages/fluentui/docs/src/examples/components/Icon/Types/IconExample.shorthand.tsx

```diff
@@ -1,12 +1,45 @@
+      </svg>
+    )}
+  </SvgIcon>
+);
+
+const callVideoSvg = ({ classes }) => (
```

Comment:
Do we really need there classes? 🤔 Can we avoid these via selectors in styles?

```
const styles = {
   root: () => ({
      // styles
      'svg': { /* styles for `svg` /* },
   })
}
```

Reply:
This also can have a performance impact, BTW

Reply:
At this moment we agreed that it's better to keep it as close as possible to the prev implementation, so we can have a smooth migration